ml-search-web/src/app/app.component.css

```diff
@@ -0,0 +1,22 @@
+.header {
+    background-color: black;
+    box-shadow: 0 0 0 0 rgba(0, 0, 0, 0), 0 0 0 rgba(0, 0, 0, 0);
```

Comment:
need to check this box shadow, this is a black shadow on a black box without any variation on any axis

Reply:
removed unnecessary styles.